tests/test_cmdline.py

```diff
@@ -465,7 +465,7 @@ def test_check__simple_project_with_extra_deps__reports_unused(
         "--check", "--detailed", "-v", f"--code={tmp_path}", f"--deps={tmp_path}"
     )
     assert output.splitlines() == expect
-    assert errors == "\n".join(expect_logs)
+    assert_unordered_equivalence(errors.splitlines(), expect_logs)
```

Comment:
Why unordered?

Reply:
Generally, I feel that our `test_cmdline` tests are getting more and more in the way of needed refactoring. One of the problems is that we have too many tests that - in addition to verifying the behavior described in the test name - also check for the existence and ordering of log messages produced by our code.

Specifically, in this case, since we now resolve _sets_ of dependency names at a time, the order in which we process each individual dependency name is no loger deterministic. Thus these two INFO log messages can show up in any order:
- `Could not find 'requests' in the current environment. Assuming it can be imported as 'requests'.`
- `Could not find 'pandas' in the current environment. Assuming it can be imported as 'pandas'.`

(There is more of this coming as part of the traverse refactoring that I talked about in today's meeting...)